packages/workbox-build/test/node/get-file-manifest-entries.js

```diff
@@ -263,38 +263,38 @@ describe('Test getFileManifestEntries', function() {
       globDirectory: path.join(__dirname, '..', '..', '..',
         'workbox-cli', 'test', 'static', 'example-project-1'),
       dynamicUrlToDependencies: {
-        '/template/url1': ['page-1.html', 'index.html'],
```

Comment:
Ideally the developer could define with and without the forward slash for the templatedURL.

The only reason I say this is because if the service worker is nested under a specific directory, you may want the URL to have a slash in some scenarios and not in others so we should support both.